satpy/writers/cf_writer.py

```diff
@@ -220,11 +337,12 @@ def _collect_datasets(self, datasets, kwargs):
             except KeyError:
                 new_datasets = [ds.copy(deep=True)]
             for new_ds in new_datasets:
-                start_times.append(new_ds.attrs.pop("start_time", None))
-                end_times.append(new_ds.attrs.pop("end_time", None))
+                start_times.append(new_ds.attrs.get("start_time", None))
```

Comment:
Now that we can encode the timestamps properly, we can keep them in the file (they contain useful information)